tasks/SPEC-one-time-better-auth-migration-cli.md

```
@@ -0,0 +1,122 @@
+pnpm dlx convex-auth migrate better-auth --resume
+
+# Limit batch/page size for large deployments.
+pnpm dlx convex-auth migrate better-auth --batch-size 100
+
+# Print help.
+pnpm dlx convex-auth migrate better-auth --help
+```
+
+## Project structure
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)